docs(how-to-guides): add instructions for sensor calibration (#145)
autowarefoundation/autoware-documentation: docs/how-to-guides/sensor-calibration.md

# How to calibrate your sensors

## Overview

> Autoware expects to have multiple sensors attached to the vehicle as input to perception, localization, and planning stack. These sensors must be calibrated correctly and the position must be defined as urdf (or as tf launch files) as in [sample_sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description).

Reviewer: Slightly reordered sentence, since current wording implies that the sample_sensor_kit uses tf launch files rather than urdf files.

```suggestion
Autoware expects to have multiple sensors attached to the vehicle as input to perception, localization, and planning stack. These sensors must be calibrated correctly and their positions must be defined using either urdf files (as in [sample_sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description)) or as tf launch files.
```

> ## Camera Calibration

Reviewer: Corrected to use sentence case.

```suggestion
## Camera calibration
```

> Navigation2 provides a good tutorial for camera internal calibraiton:
> <https://navigation.ros.org/tutorials/docs/camera_calibration.html>

Reviewer: - Typo fix and use link text instead pasting the URL.
- Would it be better to write up our own tutorial, just in case the Navigation2 documentation site disappears? 🤔 

```suggestion
Navigation2 provides a [good tutorial for camera internal calibration](https://navigation.ros.org/tutorials/docs/camera_calibration.html).
```

> ## Lidar-Lidar Calibration

Reviewer: ```suggestion
## Lidar-lidar calibration
```

TBD

> ## Lidar-Camera Calibration

Reviewer: ```suggestion
## Lidar-camera calibration
```
